test/parallel/test-fs-whatwg-url.js

```diff
@@ -30,6 +30,17 @@ fs.readFile(url, common.mustCall((err, data) => {
 // Check that using a non file:// URL reports an error
 const httpUrl = new URL('http://example.org');
 
+// Check that the fileURLToPath uses platform separator
+['file:///c:/tmp', 'file://tmp/tmp'].forEach((i) => {
```

Comment:
The problem seems to be the `file://tmp/tmp` entry here being treated as invalid. Perhaps just make it `file:///tmp/tmp`?